### ISSUE TITLE
No tables have instance_id

### ISSUE THREAD

**Issue by birdsarah** [mozilla/OpenWPM#674]
I've never found the instance_id useful, but it's still in the schemas and I've noticed it isn't in any tables from my latest crawl. If that's intentional (or maybe something went wrong with my crawl), we should clean up the schema?

**Comment by Prajwal7842** [comment]
@englehardt , Hello, I wanted to work on this issue. After going through the repo, does this issue require to remove all the statements like pa.field("instance_id", pa.uint32(), nullable=False) from different fields and responses in parquet_schema.py file ?

**Comment by vringar** [comment]
Yes, you need to remove it from the parquet_schema but also from [the S3 Aggregator](https://github.com/mozilla/OpenWPM/blob/master/automation/DataAggregator/S3Aggregator.py)
